app/models/user.rb

```diff
@@ -1,18 +1,25 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable, :registerable, and :omniauthable
   devise :rememberable, :omniauthable, omniauth_providers: [:github]
 
   validates :login, :uid, presence: true, uniqueness: true
   validates :email, :uid, presence: true, uniqueness: true
 
   def self.from_omniauth(auth)
-    where(provider: auth.provider, uid: auth.uid).first_or_create(
-      email:    auth.info.email,
-      name:     auth.info.name,
-      login:    auth.extra.raw_info.login
+    User.where(provider: auth.provider, uid: auth.uid).first_or_create(
+      email:  auth.info.email,
+      name:   auth.info.name,
+      login:  auth.extra.raw_info.login
     )
   end
+
+  def self.organisations(token)
```

Comment:
This feels wrong.
I'd say, you should just store `token` in the User model, and then this will become just a regular method on a `User` class instance: `@user.organisations`.